test/src/module/command/restoreTest.c

```diff
@@ -3392,6 +3392,109 @@ testRun(void)
 
         // Check that file was restored to full size with a partial write
         TEST_RESULT_LOG_EMPTY_OR_CONTAINS(", bi 128KB/256KB, ");
+        harnessLogLevelSet(logLevelWarn);
+
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("restore filter");
+
+        HRN_STORAGE_PUT_EMPTY(storagePgWrite(), PG_PATH_BASE "/1/40044");
+        HRN_STORAGE_PUT_EMPTY(storagePgWrite(), PG_PATH_BASE "/1/40044_fsm");
+        HRN_STORAGE_PUT_EMPTY(storagePgWrite(), PG_PATH_BASE "/1/40044_vm");
+        HRN_STORAGE_PUT_EMPTY(storagePgWrite(), PG_PATH_BASE "/1/40045");
+        HRN_STORAGE_PUT_EMPTY(storagePgWrite(), PG_PATH_BASE "/1/40045_fsm");
+        HRN_STORAGE_PUT_EMPTY(storagePgWrite(), PG_PATH_BASE "/1/40045_vm");
+        HRN_STORAGE_PUT_EMPTY(storageTest, "ts/PG_15_202209061/16416/20000");
+        HRN_STORAGE_PUT_EMPTY(storageTest, "ts/PG_15_202209061/16416/20000_fsm");
+        HRN_STORAGE_PUT_EMPTY(storageTest, "ts/PG_15_202209061/16416/20000_vm");
+        HRN_STORAGE_PUT_EMPTY(storageTest, "ts/PG_15_202209061/16416/20001");
+        HRN_STORAGE_PUT_EMPTY(storageTest, "ts/PG_15_202209061/16416/20001_fsm");
+        HRN_STORAGE_PUT_EMPTY(storageTest, "ts/PG_15_202209061/16416/20001_vm");
+        HRN_STORAGE_PUT_EMPTY(storageTest, "ts/PG_15_202209061/16416/" PG_FILE_PGVERSION);
```

Comment:
Add the `pg_filenode.map` file. To be sure that it is also restored. Because it is mentioned in the SRS.

Reply:
done